libfwupdplugin/fu-device.h

```diff
@@ -854,6 +854,8 @@ fu_device_incorporate(FuDevice *self, FuDevice *donor) G_GNUC_NON_NULL(1);
 void
 fu_device_incorporate_vendor_ids(FuDevice *self, FuDevice *donor) G_GNUC_NON_NULL(1, 2);
 void
+fu_device_incorporate_physical_id(FuDevice *self, FuDevice *donor) G_GNUC_NON_NULL(1, 2);
+void
 fu_device_incorporate_flag(FuDevice *self, FuDevice *donor, FwupdDeviceFlags flag)
```

Comment:
@superm1 the other way to do this would be to have a `fu_device_incorporate_full(FuDevice *self, FuDevice *donor, FuDeviceIncorporateFlags flags)` and then we'd have a lot more flexibility.

Reply:
_narrator_: he did that.